### PR TITLE
feat: launch mcd_chief

### PR DESCRIPTION
**Pull request by nduchak**

### CODE REVIEW

**Review by hbarcelos — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -156,6 +156,17 @@ Below is the expected structure of such a config file:
   }
 }
 ```
+NOTE: Make sure to mint enough(threshold is 8k MKR) tokens for launch if you are provide it in `imports.gov`:
+```
+    sethSend "$MCD_GOV" 'mint(address,uint256)' "$ETH_FROM" "$(seth --to-uint256 "$(seth --to-wei 1000000 ETH)")"
+```
+NOTE: Make sure to `launch` MCD_ADMIN if you are provide it in `imports.authority`.
+```
+# lock more then threshold (8k MKR)
```
Comment (lollike) [inline README.md:165]: 80,000

**Review by lollike — CHANGES_REQUESTED** [review]

**Review by hbarcelos — APPROVED** [review]

**Comment by nduchak** [comment]
Hey @gbalabasquer, can you please review this one.

File: README.md [hunk 2]
```diff
@@ -156,6 +156,18 @@ Below is the expected structure of such a config file:
   }
 }
 ```
+NOTE: Make sure to mint enough(threshold is 80,000 MKR) tokens for launch if you are provide it in `imports.gov`:
```
Comment (gbalabasquer) [inline README.md:159]:
```suggestion
NOTE: Make sure to mint enough (threshold is 80,000 MKR) tokens for launch if you are providing it in `config.gov`:
```

File: README.md [hunk 3]
```diff
@@ -156,6 +156,18 @@ Below is the expected structure of such a config file:
...
+```
+    sethSend "$MCD_GOV" 'mint(address,uint256)' "$ETH_FROM" "$(seth --to-uint256 "$(seth --to-wei 1000000 ETH)")"
+```
+NOTE: Make sure to `launch` MCD_ADMIN if you are provide it in `imports.authority`.
```
Comment (gbalabasquer) [inline README.md:163]:
```suggestion
NOTE: Make sure to `launch` MCD_ADMIN if you are providing it in `config.authority`.
```

File: README.md [hunk 4]
```diff
@@ -156,6 +156,18 @@ Below is the expected structure of such a config file:
...
+```
+# lock more then threshold (80,000 MKR)
```
Comment (gbalabasquer) [inline README.md:165]: Why more? exact threshold value is enough.
https://github.com/makerdao/dss-gov/blob/master/src/DssGov.sol#L453

File: README.md [hunk 5]
```diff
@@ -156,6 +156,18 @@ Below is the expected structure of such a config file:
...
+    sethSend "$MCD_GOV" "approve(address,uint256)" "$MCD_ADM" "$(seth --to-uint256 "$(seth --to-wei 80001 ETH)")"
```
Comment (gbalabasquer) [inline README.md:166]:
```suggestion
    sethSend "$MCD_GOV" "approve(address,uint256)" "$MCD_ADM" "$(seth --to-uint256 "$(seth --to-wei 80000 ETH)")"
```

File: README.md [hunk 6]
```diff
@@ -156,6 +156,18 @@ Below is the expected structure of such a config file:
...
+    sethSend "$MCD_ADM" "lock(uint256)" "$(seth --to-uint256 "$(seth --to-wei 80001 ETH)")"
```
Comment (gbalabasquer) [inline README.md:167]:
```suggestion
    sethSend "$MCD_ADM" "lock(uint256)" "$(seth --to-uint256 "$(seth --to-wei 80000 ETH)")"
```

File: libexec/base-deploy [hunk 7]
```diff
@@ -235,6 +235,12 @@ if [[ -z "$MCD_ADM" ]]; then
     logAddr "MCD_ADM" "$MCD_ADM"
     sethSend "$MCD_IOU" 'setOwner(address)' "${MCD_ADM}"
 
+    # lock more then threshold (80,000 MKR)
+    sethSend "$MCD_GOV" "approve(address,uint256)" "$MCD_ADM" "$(seth --to-uint256 "$(seth --to-wei 80001 ETH)")"
```
Comment (gbalabasquer) [inline libexec/base-deploy:239]:
```suggestion
    sethSend "$MCD_GOV" "approve(address,uint256)" "$MCD_ADM" "$(seth --to-uint256 "$(seth --to-wei 80000 ETH)")"
```

File: libexec/base-deploy [hunk 8]
```diff
@@ -235,6 +235,12 @@ if [[ -z "$MCD_ADM" ]]; then
...
+    sethSend "$MCD_ADM" "lock(uint256)" "$(seth --to-uint256 "$(seth --to-wei 80001 ETH)")"
```
Comment (gbalabasquer) [inline libexec/base-deploy:240]:
```suggestion
    sethSend "$MCD_ADM" "lock(uint256)" "$(seth --to-uint256 "$(seth --to-wei 80000 ETH)")"
```

**Comment by gbalabasquer** [comment]
Merged, thank you!

**Pull request merged**